cmake/version.h.in

```diff
@@ -38,6 +38,6 @@
 #cmakedefine KOVRI_CODENAME "@KOVRI_CODENAME@"
 
 #cmakedefine I2P_VERSION "@I2P_VERSION@"
-#cmakedefine I2P_NETWORK_ID "@I2P_NETWORK_ID@"
+#cmakedefine I2P_NETWORK_ID @I2P_NETWORK_ID@
```

Comment:
There was already one integer usage of it  https://github.com/monero-project/kovri/blob/master/src/core/router/i2np.cc#L53.